Moonscraper Chart Editor/Assets/Scripts/Game/UI/Inspectors/StarpowerPropertiesPanelController.cs

```diff
@@ -87,9 +90,16 @@ public void SetDrumFill()
 
         if (IsInTool())
         {
-            if (drumFillToggle.interactable)
+            if (Globals.drumMode)
+            {
+                if (drumFillToggle.interactable)
+                {
+                    SetToolFlag(drumFillToggle, Starpower.Flags.ProDrums_Activation);
+                }
+            }
+            else
             {
-                SetToolFlag(drumFillToggle, Starpower.Flags.ProDrums_Activation);
+               SetToolFlag(drumFillToggle, ~Starpower.Flags.ProDrums_Activation);
```

Comment:
Change to `SetToolFlag(drumFillToggle, Starpower.Flags.None);`, the "~" operators specifies all flags except ProDrums_Activation, so ~ProDrums_Activation is not equivalent to None.